x-pack/test/functional/page_objects/reporting_page.js

```diff
@@ -81,9 +63,8 @@ export function ReportingPageProvider({ getService, getPageObjects }) {
       `);
     }
 
```

Comment:
No longer use the response body, so we just return the raw response from here on out.